EventEnabledInsurance/post-load-test-readme.md

```diff
@@ -0,0 +1,101 @@
+<br /><br />
+
+# Prerequisites
+- Check both of the following ACE Integrations servers are ready and running:
+    - `ace-rest-int-srv-eei` - Responsible for putting messages on the MQ Queue `Quote`.
+    - `ace-db-writer-int-srv-eei` - Responsible for Reading messages from the Queue `Quote` and adding the to the Postgres Database table `db_cp4i1_sor_eei`.<br />
+    You can do so by checking the presence and status of the mentioned IntegrationServers by running the following commands:
+    ```
+    export NAMESPACE=<NAMESPACE>
+    oc get IntegrationServers -n $NAMESPACE
+    ```
+    Here the `<NAMESPACE>` is the namespace where the 1-click install initially ran.
+
+- Check that the MQ queue manager is deployed and in `Running` phase:
+    ```
+    oc get QueueManager -n $NAMESPACE mq-eei
+    ```
+- Check that the MQ queue manager pod is `Ready` and in `Running` state using the following command:
+     ```
+    oc get pods -n $NAMESPACE $(oc get pods -n $NAMESPACE | grep mq-eei-ibm-mq | awk '{print $1}')
+    ```
+- Open MQ console to check the Queue called `Quote` under Queue Manager `eei`. To get the MQ console URL using the following command:
+    ```
+    HOST=https://$(oc get routes -n ${NAMESPACE} mq-eei-ibm-mq-web -o json | jq -r '.spec.host')/ibmmq/console/
+    echo $HOST
+    ```
+    You can also access the MQ console using the Platform Navigator
+- The MQ Queue `Quote` would initially be empty.
+<br /><br />
+
+# Running the test script
+- Run the [load testing script](post-load-test.sh) with `NAMESPACE` parameter ([exported above](post-load-test-readme.md#prerequisites)) and the desired number of POST calls to be made (`TARGET_POST_CALLS` - if any value other than the default is required):
+    ```
+    export TARGET_POST_CALLS = <TARGET_POST_CALLS>
+    ./post-load-test.sh -n $NAMESPACE -c $TARGET_POST_CALLS
+    ```
+- If `-n` not provided the script will default to namespace `cp4i`.
+    At the end of the script it will display:
+    - Calls made in a second
+    - Calls failed (if any) 
+- If `-c` is not specified, the number of POST calls will default to `100`.
+<br /><br />
+
+# Checking the Queue
+- Assuming that the Integration server `ace-db-writer-int-srv-eei` is running fine, you should be able to see messages appearing disappearing from the `Quote` Queue in MQ Console on pressing the refresh icon. These messages are being added in the Postgres database table `db_cp4i1_sor_eei` by `ace-db-writer-int-srv-eei`.
```

Comment:
```suggestion
- Assuming that the Integration server `ace-db-writer-int-srv-eei` is running fine, you should be able to see messages appearing and disappearing from the `Quote` Queue in MQ Console on pressing the refresh icon. These messages are being added in the Postgres database table `db_cp4i1_sor_eei` by `ace-db-writer-int-srv-eei`.
```

Reply:
also the name fo the database would depend on the namespace name, minor detail but should probably doc that

Reply:
Added a note for that in the readme.